Sources/Logging/LogHandler.swift

```diff
@@ -171,7 +171,8 @@ extension LogHandler {
                     source: String,
                     file: String,
                     function: String,
-                    line: UInt) {
+                    line: UInt)
+    {
```

Comment:
are these formatting changes intentional?

Reply:
@tomerd The formatting updates came from the `swiftformat .` command in the `soundness` script. Any future PR updates will need to content with the either a nailed version of swift format for consistency, or update to the current version.

Reply:
reverted them -- the issue was mismatch in swift-format versions